### ISSUE TITLE
No clear way to get back to Collection Dashboard View from Edit view

### ISSUE THREAD

**Issue by julesies** [samvera-labs/nurax-pre2023#175]
### Descriptive summary
This is generally a problem with dashboard navigation but I am explaining from Collection editing perspective.

When you navigate to a collection on the dashboard, you click on the collection and then edit it, then you have no clear way to get back to the Dashboard View of that collection.  

There are 3 views of every object?
Public View
Dashboard View
Edit View

The navigation between these is pretty confusing. 

For a minimal fix, I would like to see a link back to the Dashboard View from this Edit View. 

![screen shot 2018-02-13 at 2 55 43 pm](https://user-images.githubusercontent.com/4356335/36170922-d46943e2-10ce-11e8-811e-d71ccc22c8a4.png)

**Comment by elrayle** [comment]
So you can go to the Dashboard View of the collection from the edit form by clicking Cancel.

I have advocated for that button to be Done if there are no pending changes to save and Cancel if there are pending changes to save.  But I'm not sure that is an easy fix to implement.

An easier fix to meet your desire would be to use the breadcrumbs...

Dashboard -> Collections -> View Collection -> Public View of Collection

Note that at this time, Works have only a single view.  I would like to see Works refactored to have a Dashboard View and a Public View.  It greatly helps with navigation when you spend most of your time in the Dashboard Views when you are logged in.  And it allows a logged in user to see what a public user will see based on their edits without having to log out.

**Comment by julesies** [comment]
@elrayle yeah it's weird that after you hit save, you have no clear idea what to go to next. maybe just add a "Done" or "Dashboard view" button that is a link to dashboard view?
